parser/testdata/pubsub_err_subscriber_not_function.txt

```diff
@@ -18,7 +18,7 @@ type MessageType struct {
 
 var (
     BasicTopic = pubsub.NewTopic[*MessageType]("basic-topic")
-    _ = BasicTopic.NewSubscription("basic-subscription", shared.Subscriber)
+    _ = pubsub.NewSubscription(BasicTopic, "basic-subscription", shared.Subscriber)
```

Comment:
This is out of date now I believe? We agreed on making the config required and moving the func to the config as `Handler`, I believe